packages/mjml-hero/README.md

```diff
@@ -80,24 +98,18 @@ Fluid height
 
 <p align="center">
   <a href="https://mjml.io/try-it-live/components/hero/1">
-    <img width="100px" src="https://mjml.io/assets/img/svg/TRYITLIVE.svg" alt="try it live" />
+    <img width="100px" src="https://mjml.io/assets/img/svg/TRYITLIVE.svg"
+      alt="try it live" />
   </a>
 </p>
 
 <aside class="notice">
-  The height attribute is required only for the fixed-height mode
-</aside>
-
-<aside class="notice">
-  <span style="font-weight:bold;">The background position does not work in fluid-height mode on outlook.com</span>
-</aside>
-
-<aside class="notice">
-For better result we encourage you to use a background image width equal to the hero container width and always specify a fallback background color, in case the user email client does not support background images.
+   The "height" attribute is required only for 'mode="fixed-height"'. 
 </aside>
 
 <aside class="notice">
-  Please keep the hero container height below the image height. When the hero container height - both in fixed or fluid modes - is greater than the background image height, we can’t guarantee a perfect rendering in all supported email clients
+  Outlook.com does not support 'background-position' with 
```

Comment:
I think it does now, Outlook desktop doesn't but we don't really need to give that info here